### PR TITLE
Cleanup testing

### PR DESCRIPTION
**Pull request by ralf-meyer**
Fix for issue #47, cleanup and lintering of [tests/helperFuncs.py](https://github.com/hjkgrp/molSimplify/compare/master...ralf-meyer:cleanup_testing?expand=1#diff-397b3099f4d1d6880a25788f2387ff716d389b274a8acb6937841334436dae4b) and introduces additional test cases for the xtb optimization option

### CODE REVIEW

File: tests/refs/example_3.xyz [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+H 	-0.646492	2.044971	3.072945
+H 	1.170320	2.143290	1.193163
+H 	2.160049	-0.390581	1.022001
+H 	0.404028	-2.016707	2.265259
+H 	-2.219304	-0.775337	-2.863406
+H 	-1.440798	1.785438	-2.811193
+H 	0.927166	1.934097	-1.723537
+H 	1.791302	-0.659853	-1.490443
+H 	-0.357312	-2.256132	-1.835639
 C 	-2.428227	-1.276701	1.761109
```
Comment (ralf-meyer) [inline tests/refs/example_3.xyz:23]: Shows that only one carbon atom is frozen as "connecting  atom" for this complex. Not sure if this is intended behavior.

File: molSimplify/Scripts/structgen.py [hunk 2]
```diff
@@ -817,7 +817,7 @@ def openbabel_ffopt(ff, mol, connected, constopt, frozenats, frozenangles,
 
 
 def xtb_opt(ff, mol, connected, constopt, frozenats, frozenangles,
-            mlbonds, nsteps, spin=1, debug=False):
+            mlbonds, nsteps, spin=1, inertial=False, debug=False):
```
Comment (ralf-meyer) [inline molSimplify/Scripts/structgen.py:820]: This addition might seem unrelated. It came up due to the introduction of better test coverage.

File: tests/helperFuncs.py [hunk 3]
```diff
@@ -387,7 +397,10 @@ def compare_qc_input(inp, inp_ref):
     return passQcInputCheck
 
 
-def runtest(tmpdir, name, threshMLBL, threshLG, threshOG):
+def runtest(tmpdir, name, threshMLBL, threshLG, threshOG, seed=None):
+    # Set seeds to eliminate randomness from test results
```
Comment (naveenarun) [inline tests/helperFuncs.py:401]: Nice!
